website/docs/docs/available-adapters.md

```diff
@@ -29,7 +29,8 @@ These adapter plugins are built and maintained by the same people who build and
 | ------------ | ------------- | ----------------- |
 | Materialize ([dbt-materialize](https://github.com/MaterializeInc/materialize/blob/main/misc/dbt-materialize))  | [Profile Setup](materialize-profile) | `pip install dbt-materialize` |
 | Rockset ([dbt-rockset](https://github.com/rockset/dbt-rockset))  | [Profile Setup](rockset-profile) | `pip install dbt-rockset` |
-| Starburst & Trino ([dbt-trino](https://github.com/starburstdata/dbt-trino)) | [Profile Setup](trino-profile)  | `pip install dbt-trino` 
+| Starburst & Trino ([dbt-trino](https://github.com/starburstdata/dbt-trino)) | [Profile Setup](trino-profile)  | `pip install dbt-trino` |
+| Firebolt               | [Profile Setup](firebolt-profile), [Configuration](firebolt-configs) | Firebolt | `pip install dbt-firebolt`   |
```

Comment:
This row has too many columns and is not rendering properly

Reply:
good catch, i think this is the fix
```suggestion
| Firebolt               | [Profile Setup](firebolt-profile), [Configuration](firebolt-configs) | `pip install dbt-firebolt`   |
```

Reply:
Nice. Can we link to the repo like the others do?